libs/fetcher.js

```diff
@@ -303,7 +318,10 @@ Fetcher.middleware = function () {
                         res.status(errResponse.statusCode).json(errResponse.output);
                         return;
                     }
-                    res.status(meta.statusCode || 200).json(data);
+                    res.status(meta.statusCode || 200).json({
```

Comment:
So this could break BC if the client hasn't refreshed the page but the server gets updated to latest.

So I'm going to put in a feature flag to control this behavior and we can remove it in future versions.